tests/4.5/taskloop/test_taskloop_firstprivate.c

```diff
@@ -0,0 +1,63 @@
+//----------------------------------- test_taskloop_firstprivate.c ----------------------------------------//
+//
+// OpenMP API Version 4.5 November 2018
```

Comment:
Change 2018 to 2015 since it is an OpenMP 4.5 test now.

Reply:
done